README.md

```diff
@@ -0,0 +1,10 @@
+# Hi! I'm Ben. Nice to meet you.
+> Web Developer Greenville, SC
+
+## About Me
+I am a proud Greenville resident along with my beautiful wife and daughter. As a creative problem solver, 2020 gave me the opportunity to start a new path in Web Development and I'm excited to use my skills to help make other people's lives better.
+
+I'd love to have a conversation. Please reach out and let me know where you'd like to get a drink!
+
+## About this site
+<!-- Ben you should talk about how you built your site here. -->
```

Comment:
@benhungerford I added a Readme so your site repo will look nice on Github! let me know if you have any questions.